OpenActive.Server.NET/StoreBookingEngine/StoreBookingEngine.cs

```diff
@@ -586,8 +586,8 @@ public async override Task<TOrder> ProcessFlowRequest<TOrder>(BookingFlowContext
                                         throw new OpenBookingException(new InternalLibraryError(), "SetOrderItemId must be called for each OrderItemContext in ProposeOrderItems");
                                     }
 
-                                    // Set the orderItemStatus to be https://openactive.io/OrderItemProposed (as it must always be so in the response of P)
```

Comment:
Why null instead of `OrderItemProposed` ?

Reply:
Good question: https://github.com/openactive/open-booking-api/issues/157